app/store/store.js

```diff
@@ -0,0 +1,15 @@
+import { configureStore } from '@reduxjs/toolkit'
+import rackings from './reducers/rackings'
+import systems from './reducers/rackings'
+
+const reducer = {
+  rackings,
+  systems,
+}
+
+const store = configureStore({
+  reducer,
+  middleware: (getDefaultMiddleware) => getDefaultMiddleware(),
```

Comment:
I'm guessing you are waiting until you start using thunks before adding it?

Reply:
Yeah, going to get the api sauce set up beforehand